.github/workflows/functional-test.yaml

```diff
@@ -33,31 +34,31 @@ on:
   repository_dispatch:
     types: [de-functional-test]
   workflow_run:
-    workflows: ['Approve Functional Tests']
+    workflows: ["Approve Functional Tests"]
     types:
       - completed
 
 env:
   # Go version
-  GOVER: '1.22.2'
+  GOVER: "1.22.2"
   GOPROXY: https://proxy.golang.org
 
   # gotestsum version - see: https://github.com/gotestyourself/gotestsum
-  GOTESTSUM_VER: 1.10.0
+  GOTESTSUM_VER: 1.12.0
```

Comment:
It's been more than a year since the 1.10.0 version was released so I updated it: https://github.com/gotestyourself/gotestsum/releases.